schemas/Account.json

```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Admin users may temporarily disable an account by setting this flag",
```

Comment:
"temporarily" makes it sound like it *cant* be permanent, but this is potentially permanent, but reversible right?